frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-items.tsx

```diff
@@ -534,6 +560,12 @@ export const OperatorHubTileView: React.FC<OperatorHubTileViewProps> = (props) =
   );
 };
 
+type OperatorHubTileProps = {
+  item: OperatorHubItem;
+  onClick: (item: OperatorHubItem) => void;
+  t: TFunction;
```

Comment:
You don't typically want to pass `t` as a prop. Not all `t` functions are the same since `useTranslation` takes parameters. Better to call `useTranslation` in each component that needs it.